openassessment/xblock/static/js/src/lms/oa_response.js

```diff
@@ -35,6 +35,21 @@ OpenAssessment.ResponseView.prototype = {
     // Maximum file size (5 MB) for an attached file.
     MAX_FILE_SIZE: 5242880,
 
+    UPLOADABLE_IMAGE_MIME_TYPES: [
+        'image/gif',
+        'image/jpeg',
+        'image/pjpeg',
+        'image/png',
+    ],
+
+    UPLOADABLE_FILE_MIME_TYPES: [
+        'application/pdf',
+        'image/gif',
+        'image/jpeg',
+        'image/pjpeg',
+        'image/png',
+    ],
```

Comment:
My javascript isn't great - is there a way to define `UPLOADABLE_FILE_MIME_TYPES` as basically (as you'd do in Python), `UPLOADABLE_FILE_MIME_TYPES = UPLOADABLE_IMAGE_MIME_TYPES + ['application/pdf']`?

Then there's less work to keep them in sync.